src/GraphQL/Execution/ErrorInfoProviderOptions.cs

```diff
@@ -10,10 +10,25 @@ public class ErrorInfoProviderOptions
         /// <summary>
         /// Specifies whether stack traces should be serialized.
         /// </summary>
-        public bool ExposeExceptionStackTrace { get; set; }
+        [Obsolete("Use ExposeExceptionDetails property instead")]
+        public bool ExposeExceptionStackTrace
+        {
+            get => ExposeExceptionDetails;
+            set => ExposeExceptionDetails = value;
```

Comment:
```suggestion
            set
            {
                ExposeExceptionDetails = value;
                if (value)
                    ExposeExceptionDetailsMode = Message;
            }
```
Just an idea.....

Reply:
> do you think Message should be the default mode?

for v5.4 yes, for v7 I would like to switch it to Extensions and remove `ExposeExceptionStackTrace`

Reply:
I don't see any good reason to remove the member now, immediately before v7 is released.  It can be a deprecated member for v7.

I also would not switch the default of the new property between versions.

I would do as I posted above -- setting the old property retains the old behavior, while setting the new property defaults to the new behavior.  There is no breaking change for 5.4 and no breaking change when 7 is released.  The [Obsolete] comment needs to be updated to reflect this of course.

Reply:
ExposeExceptionDetailsMode defaults to `Message`. Why to reset it to Message again into setter ?

Reply:
ExposeExceptionStackTrace is the same thing as new property. It is just a way to not renaming it and introduce breaking changes. I don't want to bring some special behavior into setter.

Reply:
The goal is to eventually remove `ExposeExceptionStackTrace` and _maybe_ change default mode from Message to Extensions. 

I'm done for today, too tired to think more.

Reply:
> ExposeExceptionDetailsMode defaults to Message. Why to reset it to Message again into setter ?

You said you wanted Extension to be the default, which I agree with.

> I don't want to bring some special behavior into setter.

Then you can't release this in 5.4.  It must be v7.  If you release it in 5.4 then it will cause a breaking change that is not marked by an obsolete flag in v7.  There is no reason to do so with v7 so near to being released.

Either that or Message should continue to be the default after v7 is released.

Or you go with my 3rd option, where using the new option uses the new default, and using the old option uses the old default.

Reply:
I really don't see why not to do as I suggested.  It allows for non-API-breaking behavior in both v5.4 and v7, while having the new default value of 'Extension' for all new code.  Then we can remove the property in v8.

>  I don't want to bring some special behavior into setter.

Nor do I, but in this case, it is clearly the best choice for the scenario.  (in my opinion)

Reply:
> The goal is to eventually remove `ExposeExceptionStackTrace` and _maybe_ change default mode from Message to Extensions.

If there is no goal to either change the default from Message or to remove the old property in v7, then we can merge as-is.  We should think about it and decide one way or the other.

Reply:
I was busy again. Now I looked at the code again and realized that the difference was not fundamental to me at all. As a result, the old property will eventually be deleted, and now I will not use it. I applied suggestion.